tests/jekyll-build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eux
+sudo gem install github-pages bundler
+bundle update
+bundle exec jekyll build
+mv _site logs
```

Comment:
We expose the log directory via $BONNYCI_TEST_LOG_DIR, we should be using that here in case we update our publishers in the future.